.github/workflows/binaries.yml

```diff
@@ -350,12 +361,12 @@ jobs:
         shell: bash
         run: |
           cd jabgui/build/distribution
-          ar x jabref_${{ steps.gitversion.outputs.Major }}.${{ steps.gitversion.outputs.Minor }}${{ matrix.suffix }}_amd64.deb
+          ar x jabref_${{ steps.gitversion.outputs.Major }}.${{ steps.gitversion.outputs.Minor }}${{ matrix.suffix }}.deb
```

Comment:
I had the same error as well and I think it's missing the arch a second time because on arm I had then something like _am64_arm64